### ISSUE TITLE
How can I string prompts to create a login?

### ISSUE THREAD

**Issue by JitterDev** [enquirer/enquirer#320]
I want to create a simple login using 2 prompts with Select and BasicAuth (if there are betters prompts to use for this, Please tell me.) Select handles the username. I want the answer of Select to tell BasicAuth to use "answer"Key from my configuration file. My configuration file looks like this:
`module.exports = {`
    `bobKey: '',`
    `lisaKey: '',`
    `johnKey: '',`
`};`
My main code looks like this:
`const { Select } = require('enquirer');`
`const { BasicAuth } = require('enquirer');`
`const express = require('express')`
`const app = express()`
`const config = require('./config')`

`const usernameSelect = new Select({`
    `name: 'login',`
    `message: 'Select your username',`
    `choices: ['bob', 'lisa', 'john']`
`});`

`usernameSelect.run()`
`.then(answer => )`

**TL;DR**
How do I make my answer of prompt 1 be config. _answer_ Key for the value of a basicauth prompt?

Sorry if this was confusing to explain. My code is kind of messy

**Comment by JitterDev** [comment]
Think I found a solution, will make another issue/reopen if I have problems
